run-checks.sh

```diff
@@ -108,7 +108,8 @@ function main() {
 
   _test_wrapper "YAML style" "_docker osu-wiki-tools check-yaml --target" .
 
-  _test_wrapper "link" "_docker osu-wiki-tools check-links --target" "${interesting_articles}"
+  # TODO: _test_wrapper expects a third argument but luckily --all overrides --target
+  _test_wrapper "link" "_docker osu-wiki-tools check-links --all --target" "${interesting_articles}"
```

Comment:
if the intention is to rework `_test_wrapper` so that it supports two arguments instead of three, or either two or three, it's better to say that explicitly rather than state a fact

Reply:
```suggestion
  # TODO: _test_wrapper expects a third argument but luckily --all overrides --target. please rewrite to make this prettier
  _test_wrapper "link" "_docker osu-wiki-tools check-links --all --target" "${interesting_articles}"
```